pattern/constraint/type/LabelConstraint.java

```diff
@@ -104,7 +104,8 @@ public String toString() {
 
     @Override
     public FunctionalIterator<AlphaEquivalence> alphaEquals(LabelConstraint that) {
-        return AlphaEquivalence.empty().alphaEqualIf(label().equals(that.label()));
+        return AlphaEquivalence.empty().alphaEqualIf(label().equals(that.label()))
```

Comment:
tbh i think it would be nice and more consistent if you always check the owner first

Reply:
Nice, I've switched it.